src/components/tx/SignOrExecuteForm/SignOrExecuteForm.test.tsx

```diff
@@ -572,7 +572,7 @@ describe('SignOrExecuteForm', () => {
     expect(proposeSpy).toHaveBeenCalledTimes(1)
   })
 
-  it('smart contract wallets have to propose when creating a tx with an on-chain signature', async () => {
+  xit('smart contract wallets have to propose when creating a tx with an on-chain signature', async () => {
```

Comment:
Why are we skipping this now?

Reply:
SC wallets are not signing on chain in this branch but i was unsure if should remove the test. Can we discuss this with @schmanu ?

Reply:
I don't think this PR touches the on-chain signing modal. So this test should still work.

In general I would not remove all ability for on-chain signing. It is not decided yet how we go forward with deciding which signing method to use.